src/Ucommerce.Sitecore.Cli/Commands/Database.cs

```diff
@@ -7,47 +9,58 @@
 
 namespace Ucommerce.Sitecore.Cli.Commands
 {
-    [Command("db")]
-    public class Database : ICommand
+    /// <summary>
+    /// Base class for db commands exposing shared Options
+    /// </summary>
+    public abstract class Db
     {
-        [CommandParameter(0, Description = "Connection string for the database.")]
+        [CommandOption("ConnectionString",
```

Comment:
Should this be CommandParameter instead of CommandOption, since it is not an optional parameter?